test/fixtures/mod-deps-vendored/main.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/gorilla/mux"
```

Comment:
Hmm...interesting. How did this test pass? Without specifying `GO111MODULE=on`, Go1.11rc2 doesn't fetch deps for me.

Reply:
NVM, you vendored all deps for this fixture.